snoop/emails.py

```diff
@@ -95,6 +96,12 @@ def open_part(self, number):
             data = part.get_payload(decode=True)
         except:
             raise PayloadError
+
+        if "encrypted" in self.flags and \
```

Comment:
It's also used (along with the warnings) when get_part_text() fails.
How about the warnings logic? Do I get to nuke that too?